src/vivarium_census_prl_synth_pop/components/businesses.py

```diff
@@ -379,3 +358,12 @@ def calculate_income(self, idx: pd.Index) -> pd.Series:
         )
 
         return income
+
+    def get_business_details(self, idx: pd.Index) -> pd.DataFrame:
+        """Source of the business details pipeline"""
+        pop = self.population_view.get(idx)[["employer_id"]]
```

Comment:
That's because we actually need to record them for untracked simlants in the tax observers, whereas we don't need income for those observers, since they use annual earnings on observation rather than income.